src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The interface name 'AreaCalculator' should not be a verb. Consider renaming it to something that represents an entity or a characteristic, such as 'CalculableArea' or 'AreaComputable'.